### DISCUSSION TITLE
How to switch between real and fake screens?

### DISCUSSION THREAD

**Discussion by Nebucatnetzer** [qtile/qtile#4445] [Q&A] [answered]
I have a draft for fake_screens in order to split my monitor into four Full HD screens.
https://git.2li.ch/Nebucatnetzer/nixos/src/commit/cdaab04a87e019b7e8e32260d3a1d31ac7d15e07/modules/services/qtile/config.py#L316

So far it works, however what I haven't figured out yet, is to switch between real and and fake screens with a keyboard shortcut.
In addition I don't know that happens to the second real monitor when I activate the fake screens on the main monitor.

The main goal is to have the primary monitor split into fake screens and have additional physical monitors connected which keep their normal configuration.

**Comment by m-col** [comment]
This isn't possible I'm afraid. However it seems that in x11 you can get the X server to report a physical screen as multiple outputs, which may get the desired effect. See https://github.com/phillipberndt/fakexrandr

**Comment by Nebucatnetzer** [comment]
Thank you for the response. I don’t use the option often so I could probably work without the second physical monitor.
What about switching between the fake- and realscreen configuration with a keyboard shortcut? Is that possible?

**Reply by m-col** [reply, marked as answer]
This isn't possible sorry. Depending on your workflow you may be interested in https://github.com/qtile/qtile/pull/3793

**Reply by Nebucatnetzer** [reply]
Okay thank you, I'm subscribed to that issue.
Looking forward to it, the feature is really great.
